package-lock.json

```diff
@@ -1,13126 +1,8 @@
 {
     "name": "vscode-azurefunctions",
     "version": "1.4.2-alpha",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
Looks like your npm version is out of date. Please update (I'm on v7.19.1) and make sure there are minimal changes to this file

Reply:
Pushed a new version with updated npm